src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,71 @@
+package task;
+
+import components.Parser;
+import exception.NyabotParseException;
+
+import java.util.ArrayList;
+
+
+public class TaskList {
```

Comment:
Perhaps you could add JavaDoc header comments for this class file? I noticed that most of the other classes have header comments but this class's was missing